### PR TITLE
#81 verify jobs

### PR DESCRIPTION
**Pull request by bio-boris**
#81 verify jobs

### CODE REVIEW

**Comment by bio-boris** [comment]
I ran this script against CI and it didn't throw any errors, so I think this issue is fixed. Unless my script needs some revisions.

File: lib/execution_engine2/db/models/transfer/check_jobs_for_finish_times.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        valid_status(status)
+    except:
+        raise Exception(job_id)
+    if count % 500 == 0:
+        print(f"Processed {count} records")
+
+    if status in [Status.error.value, Status.completed.value, Status.terminated.value]:
+        end_time = job.get("finished")
+        if end_time is None:
+            raise Exception("End time is NONE", job_id)
```
Comment (Tianhao-Gu) [inline lib/execution_engine2/db/models/transfer/check_jobs_for_finish_times.py]: do we want to raise here or return all bad job ids and fix them in a batch?

Comment (bio-boris) [reply lib/execution_engine2/db/models/transfer/check_jobs_for_finish_times.py]: I think raise is good so the script crashes and I can just start looking at the database. That's what I did already  and fixed the jobs below

Comment (Tianhao-Gu) [reply lib/execution_engine2/db/models/transfer/check_jobs_for_finish_times.py]: 👍

**Pull request merged**